Automattic/wp-calypso
# Virtual themes: Remove description

## Proposed Changes

In https://github.com/Automattic/wp-calypso/pull/74210, we started showing a description for virtual themes in order to give users an expectation about what theme will be activated in the site.

This is now clearly visible in the design title now that virtual themes include the theme name (D107549-code), so we no longer need this description.

Before | After
--- | ---
<img width="1278" alt="Screenshot 2023-04-11 at 17 23 57" src="https://user-images.githubusercontent.com/1233880/231213492-224f8666-d9bb-4b4b-9031-a9f24b6fd58a.png"> | <img width="1275" alt="Screenshot 2023-04-11 at 17 24 34" src="https://user-images.githubusercontent.com/1233880/231213516-1037b40c-49f1-4326-87b3-5fffadecc7a6.png">


This also makes the preview screen more consistent since now all themes without style variations (virtual and non-virtual) won't display the description sidebar.

Someone could argue that it'd be nice to keep the Pattern Assembler CTA but it won't have much value once we can start sending users to PA to further customize assembled designs.

## Testing Instructions

- Use the Calypso live link below.
- Go to `/setup/site-setup/designSetup?siteSlug=<SITE_DOMAIN>`.
- Select a virtual design.
- Make sure there is no sidebar.
